pkg/sync/v6/sync.go

```diff
@@ -43,6 +43,7 @@ const (
 	ownedBySyncPodLabelKey            = "azure.openshift.io/owned-by-sync-pod"
 	syncPodWaitForReadinessLabelKey   = "azure.openshift.io/sync-pod-wait-for-readiness"
 	syncPodReadinessPathAnnotationKey = "azure.openshift.io/sync-pod-readiness-path"
+	deployOnceSyncPodLabelKey         = "azure.openshift.io/deploy-once"
```

Comment:
it's an AnnotationKey

Reply:
the deploy-once name is somewhat edge-triggered - is it possible to find an level-triggered name?

Reply:
I might need an explanation what do you have in mind?